docs/_toc.yml

```
@@ -16,6 +16,9 @@ chapters:
   sections:
   - file: tutorials/01_start
   - file: tutorials/02_speed
+  - file: tutorials/03_emulation_sensitivity
+  - file: tutorials/04_active_learning
+  - file: tutorials/05_reaction_diffusion
 
```

Would it be worth adding the second notebook here too?
```suggestion
  - file: tutorials/05_reaction_diffusion
  - file: tutorials/06_reaction_diffusion_time
```

---

In our meeting last week we decided to leave it from the docs for now until we can say more about explicitly handling the time dimension. But we're merging the tutorial so that it is available for when we get to that point.

---

I have refined the tutorial 5 but not tutorial 6 yet, so I thought about putting on the website once it is finalised

---

Or should I just add it right now anyways and refine it later on?